src/components/BackButtonRibbon.vue

```diff
@@ -4,7 +4,10 @@
       class="back-button"
```

Comment:
Then the flex layout will not be correct any more

Reply:
Yes, I see that. Would be clearer to replace the button with an empty element (like `<div v-else />`) instead of hiding the button content as now. But, it is ok to leave it as it is now.